crates/sled/src/tree.rs

```diff
@@ -198,12 +198,28 @@ impl Tree {
     /// assert_eq!(t.del(&[1]), Ok(Some(sled::IVec::from(vec![1]))));
     /// assert_eq!(t.del(&[1]), Ok(None));
     /// ```
+    #[deprecated(since = "0.24.2", note = "replaced by `Tree::remove`")]
```

Comment:
I chose `0.24.2` but it is up to you to tell me the version where the deprecation begins.

Reply:
That's good! The next version to be published will either be 0.25 or 0.666 depending on if I want to do one before beta.